app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -23,8 +23,33 @@ class ReportbackController extends ApiController
     public function __construct(ReportbackService $reportbackService)
     {
         $this->reportbackService = $reportbackService;
-        $this->transformer = new ReportbackTransformer;
-        $this->itemTransformer = new ReportbackItemTransformer;
+        $this->transformer = new PhoenixGalleryTransformer;
```

Comment:
I feel like this could just be `ReportbackTransformer` as we don't want name transformers based on how they are going to be used, but more like what model they are transforming.